ci/pipelines/resources/template.jsonnet

```
@@ -148,7 +148,7 @@ local create_release = {
 };
 
 local publish_job(bump) = {
-  name: bump,
+  name: "Publish " + bump,
```

Hey, what about going with the same pattern we use for the main pipeline?

```suggestion
  name: "publish-" + bump,
```

https://github.com/concourse/concourse/blob/d64843ce1bbab2309213ff677e25aadb950fc33b/ci/pipelines/concourse.yml#L79-L81